### ISSUE TITLE
use example based approach to  Codelab Formatting Guide

### ISSUE THREAD

**Issue by djschny** [googlecodelabs/tools#30]
This is in reference to the [Codelab Formatting Guide](https://docs.google.com/document/d/18dnMdUJQaGKY1Tit_-fO1YOpOpAbA4hh0YDXQlCEjvA/edit).

Similar to the [Examples section](https://guides.github.com/features/mastering-markdown/#examples) of the Github Markdown guide, having examples people can just copy paste and then modify with their content I feel would be much more productive than instructing them on how to do formatting.

That way folks can just scan through the document and when they see something that looks like what they want to use, they just copy the content from the formatting guide Google Doc into their Lab Doc and modify/tweak.

My apologies if this has already been discussed and I messed it when searching.

**Comment by x1ddos** [comment]
@djschny do you mean examples on a web page, translated from a markdown file or this gihub repo wiki?
Otherwise, the doc you linked to does provide some examples.

**Comment by x1ddos** [comment]
Actually, maybe it's about time we translate that doc into a wiki page in this repo.
@marcacohen wdyt?

**Comment by djschny** [comment]
> @djschny do you mean examples on a web page, translated from a markdown file or this gihub repo wiki?

I'm referring to the linked Codelab Formatting Guide google doc.

> Otherwise, the doc you linked to does provide some examples.

Sadly upon further inspection, when reviewing the doc to produce some labs, I completely missed the `For example:` prior to the actual example in each doc. My mind transposed the example and screenshot together.

My sincere apologies for not noticing. Feel free to close the issue, unless you want to leave it open for the last comment about translating to a wiki page in this repo.

Thanks for taking the time to look at this and sorry about the noise.

**Comment by marcacohen** [comment]
It'd be nice to have a wiki page. The advantage of doing that is we won't have a separate doc on which to manage sharing. And it'll be easier for people to access. I'll do it.

**Comment by x1ddos** [comment]
Awesome. Thanks @marcacohen!

**Comment by x1ddos** [comment]
Hey Marc, any progress here? I'm thinking maybe instead of a wiki page, we might want just a markdown file so that people can propose changes and improvements via pull requests.

**Comment by marcacohen** [comment]
I agree, md is better than wiki for this purpose. Leave this assigned me for now (have a few higher prio items atm but I'll get to it if no one else wants to take it off my hands).

**Comment by samtstern** [comment]
Even better: the codelab formatting guide should be a codelab!  This is what Ubuntu does:
https://tutorials.ubuntu.com/tutorial/tutorial-guidelines#0

Bonus points if the source for the codelab is in markdown, which makes it something you can copy-paste from and self-proving.

**Comment by helmutgranda** [comment]
Would it have to be a 1:1 with the following document?

https://docs.google.com/document/d/18dnMdUJQaGKY1Tit_-fO1YOpOpAbA4hh0YDXQlCEjvA

If that is the case the conversion should be rather quickly.
